### PR TITLE
Always resolve symlink when using os.Executable

### LINKED ISSUE #2700: [BUG] Bundle not extracted on macos installer
On pre release bit 1.32.0 the macos installer does not extract the bundle to the default location, then `setup` will try to uncompress from this location and ends up failing;

`crc version`:
```
macmini-crcqe-1:1.32.0 crcqe$ crc version
CodeReady Containers version: 1.32.0+655578ee
OpenShift version: 4.8.5 (bundle installed at /Users/crcqe/.crc/cache/crc_hyperkit_4.8.5.crcbundle)
```

`crc setup` log:
```
macmini-crcqe-1:1.32.0 crcqe$ crc setup --log-level debug
DEBU CodeReady Containers version: 1.32.0+655578ee 
DEBU OpenShift version: 4.8.5 (bundle installed at /Users/crcqe/.crc/cache/crc_hyperkit_4.8.5.crcbundle) 
DEBU Running 'crc setup'                          
INFO Checking if running as non-root              
INFO Checking if crc-admin-helper executable is cached 
INFO Checking for obsolete admin-helper executable 
DEBU Checking if an older admin-helper executable is installed 
DEBU No older admin-helper executable found       
INFO Checking if running on a supported CPU architecture 
INFO Checking minimum RAM requirements            
DEBU Total memory of system is 34359738368 bytes  
INFO Checking if running emulated on a M1 CPU     
INFO Checking if HyperKit is installed            
INFO Checking if qcow-tool is installed           
INFO Checking if crc-driver-hyperkit is installed 
INFO Checking if CodeReady Containers daemon is running 
DEBU Running 'bash -c launchctl list | grep crc.daemon | awk '{print $1}'' 
INFO Checking if launchd configuration for tray exists 
INFO Check if CodeReady Containers tray is running 
DEBU Running 'bash -c launchctl list | grep crc.tray | awk '{print $1}'' 
DEBU Tray is not running                          
INFO Starting CodeReady Containers tray           
DEBU Starting tray agent                          
DEBU Running 'launchctl load /Users/crcqe/Library/LaunchAgents/crc.tray.plist' 
DEBU Running 'launchctl start crc.tray'           
INFO Checking if CRC bundle is extracted in '$HOME/.crc' 
INFO Checking if /Users/crcqe/.crc/cache/crc_hyperkit_4.8.5.crcbundle exists 
DEBU error getting bundle info for crc_hyperkit_4.8.5.crcbundle: could not find cached bundle info in /Users/crcqe/.crc/cache/crc_hyperkit_4.8.5: stat /Users/crcqe/.crc/cache/crc_hyperkit_4.8.5: no such file or directory 
DEBU could not find cached bundle info in /Users/crcqe/.crc/cache/crc_hyperkit_4.8.5: stat /Users/crcqe/.crc/cache/crc_hyperkit_4.8.5: no such file or directory 
INFO Extracting bundle from the CRC executable    
INFO Ensuring directory /Users/crcqe/.crc/cache exists 
INFO Uncompressing crc_hyperkit_4.8.5.crcbundle   
DEBU Uncompressing /Users/crcqe/.crc/cache/crc_hyperkit_4.8.5.crcbundle to /Users/crcqe/.crc/cache/tmp-extract 
open /Users/crcqe/.crc/cache/crc_hyperkit_4.8.5.crcbundle: no such file or directory
```

### PR DESCRIPTION
**Pull request by guillaumerose**
Fix #2700

### CODE REVIEW

File: pkg/crc/version/version.go [hunk 1]
```diff
@@ -91,7 +91,12 @@ func InstallPath() string {
 		logging.Errorf("Failed to find the installation path: %v", err)
 		return ""
 	}
-	return filepath.Dir(currentExecutablePath)
+	src, err := filepath.EvalSymlinks(currentExecutablePath)
+	if err != nil {
+		logging.Errorf("Failed to find the symlink destination of %s: %v", currentExecutablePath, err)
+		return ""
```
Comment (cfergeau) [inline pkg/crc/version/version.go:97]: might be better to return `filepath.Dir(currentExecutablepath)`?

**Review by cfergeau — APPROVED** [review]
looks good to me!

**Review by anjannath — APPROVED** [review]
Tested and works for me on windows!

File: pkg/crc/version/version.go (hunk 1 above)
Comment (guillaumerose) [reply pkg/crc/version/version.go]: done!

**Pull request merged**